tgui/packages/tgui/styles/interfaces/DirectorPanel.scss

```
@@ -0,0 +1,109 @@
+  color: rgba(255, 255, 255, 0.55);
+  font-size: 0.9em;
+  line-height: 1.25;
+}
+
+.DirectorPanel__MetricBar {
+  margin-top: 0.2em;
+}
+
+.DirectorPanel__StatusTag {
+  padding: 0.08em 0.45em;
+  border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)